ports/windows/README.md

```diff
@@ -5,9 +5,12 @@ consider to contribute.
 
 
 Building on Debian/Ubuntu Linux system
+Building on Windows 10 with WSL (Windows Subsystem for Linux) Ubuntu Bash shell
```

Comment:
Just noticed that if you look at this file rendered on Github this looks meesed up as it becomes one line of bold text, so better reword it.